recipes/libssh2/all/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8.12)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
@theirix did you change it because a hook error? If yes, it's a bug.

Reply:
yes, the hook does not check for whitespace

Reply:
Is it valid CMake to have there whitespaces? 🤔

Reply:
Yes, it matches grammar and works well. Also it workes even with such ancient CMake version.

Reply:
okay, I'll send a patch to fix it.